src/server/config.rs

```diff
@@ -232,7 +232,7 @@ impl Default for Config {
             ),
             end_point_max_concurrency: cmp::max(cpu_num as usize, MIN_ENDPOINT_MAX_CONCURRENCY),
             snap_max_write_bytes_per_sec: ReadableSize(DEFAULT_SNAP_MAX_BYTES_PER_SEC),
-            snap_max_total_size: ReadableSize(0),
+            snap_max_total_size: ReadableSize(50 * GIB),
```

Comment:
Can you explain why it's `50GiB`?

Reply:
The size consideration is that:
1) It should be reasonable big enough so that it can have a few snapshot files being generated at parallel. 
2) But it should not too big otherwise it would waste disk space when snapshots fail to be sent.